src/openloco/scenario.cpp

```diff
@@ -1,12 +1,55 @@
 #include "scenario.h"
+#include "date.h"
 #include "graphics/gfx.h"
 #include "interop/interop.hpp"
 #include "ui/WindowManager.h"
+#include "ui/WindowType.h"
 
 using namespace openloco::interop;
 
 namespace openloco::scenario
 {
+    // Inflation rates?
+    static const uint32_t _4FDEC0[32] = {
+        20,
+        20,
+        20,
+        20,
+        23,
+        20,
+        23,
+        23,
+        20,
+        17,
+        17,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+        20,
+    };
+
+    static loco_global<uint32_t[32], 0x00525E5E> currencyMultiplicationFactor;
```

Comment:
Assuming these values start at `1`, this is fixed point with 10 fractional bits.